### DISCUSSION TITLE
Error on upload file to a springboot API using Axios

### DISCUSSION THREAD

**Discussion by JSsantana** [remix-run/remix#2972] [Q&A]
Error result on the API: 
`org.apache.tomcat.util.http.fileupload.FileUploadException: the request was rejected because no multipart boundary was found`

My Action: 

```
export const action: ActionFunction = async ({ request }) => {
  const uploadHandler = unstable_createFileUploadHandler({
    file: ({ filename }) => filename,
  });

  const formData = await unstable_parseMultipartFormData(
    request,
    uploadHandler
  );

  const file = formData.get("dataSheet");
  const restEndpoint = "my-end-point";

  try {
    await axios({
      method: "post",
      url: restEndpoint,
      data: file,
      headers: { "Content-Type": "multipart/form-data" },
    }).then((res) => console.log(res));
  } catch (error) {
    console.log(error);
  }
  return null;
};

```

Tried to remove "Content-Type": "multipart/form-data" from headers but got another error:
`Current request is not a multipart request`

my form element:
  `<Form method="post" encType="multipart/form-data"></Form>`

**Comment by cstrnt** [comment]
Instead of passing in the File itself you could simply pass in the `formData` and send this using axios
